packages/jest/src/schematics/jest-project/jest-project.ts

```diff
@@ -114,6 +114,10 @@ function check(options: JestProjectSchema): Rule {
 }
 
 function normalizeOptions(options: JestProjectSchema): JestProjectSchema {
+  if (options.testEnvironment === 'jsdom') {
+    options.testEnvironment = '';
+  }
```

Comment:
The tests that were there prior cover this as the cli flag will default to jsdom which means this if is true and testEnvironment is an empty string then when the jest.config.js is generated if testenvironment is falsy it is left out (as jsdom is default for jest). I can put a test in explicity for this if you would like but because the tests that were there before pass we know that this works. Let me know if an explicity test is needed.

Reply:
Well e2e test more for the flag, setting the property and making sure that all the options are covered in the output.

Reply:
Sounds good.